tests/integration/test_fim/test_files/test_prefilter_cmd/test_prefilter_cmd_conf.py

```diff
@@ -58,17 +59,19 @@ def check_prelink():
 @pytest.mark.parametrize('tags_to_apply', [
     ({'prefilter_cmd'})
 ])
```

Comment:
Only one configuration is used, this parametrized is not necessary. Also rename `yaml` in order to fit better with the test name

Reply:
Fixed in 0d4b09f2d4fa3ebbe631518a23861d1a79517f96